### PR TITLE
[Enhancement] modify comment " to ' compat mysql.

### LINKED ISSUE #10203: [Enhancement] Show create table Comment use single quotes
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Description

Current show create table use of double quotes, which is not standard mysql syntax, causes some BI platforms to fail to resolve comments, such as Superset.

### Solution

  `name` varchar(100) NULL COMMENT "姓名"  ->   `name` varchar(100) NULL COMMENT '姓名'

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by stalary**
# Proposed changes

Issue Number: close https://github.com/apache/doris/issues/10203

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/test/java/org/apache/doris/analysis/ShowCreateTableStmtTest.java [hunk 1]
```diff
@@ -18,46 +18,36 @@
...
-    private PaloAuth auth;
-    @Mocked
-    private ConnectContext ctx;
-
-    @Before
-    public void setUp() {
-        analyzer = AccessTestUtil.fetchAdminAnalyzer(true);
-        MockedAuth.mockedAuth(auth);
-        MockedAuth.mockedConnectContext(ctx, "root", "192.168.1.1");
+import org.apache.doris.common.DdlException;
```
Comment (jackwener) [inline fe/fe-core/src/test/java/org/apache/doris/analysis/ShowCreateTableStmtTest.java:21]: Use `control+option+o` remove unused header.
[Doc about it](https://doris.apache.org/zh-CN/developer/developer-guide/java-format-code.html#import-order)

Comment (stalary) [reply fe/fe-core/src/test/java/org/apache/doris/analysis/ShowCreateTableStmtTest.java:21]: Ok, it is still in the development stage, I will deal with it later.

**Comment by stalary** [comment]
I found that this did not solve the superset presentation problem and that the change affected many UT and Regression, so I close it.

**Comment by stalary** [comment]
superset and dathub use sqlalchemy by `show create table` sql get metadata，Standard mysql syntax, comment uses single quotes.

**Review by morningman — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]

**Pull request merged**